### ISSUE TITLE
Couldn't train text encoder with Dreambooth XL

### ISSUE THREAD

**Issue by giteeeeee** [bmaltais/kohya_ss#1882]
Tried training a XL model under dreambooth, but couldn't get the text encoder to train:

![123](https://github.com/bmaltais/kohya_ss/assets/89521447/3dbc31b0-cc20-484a-893a-0fcd6bc07ca6)

Then you may ask how do I know the text encoder is not actually training? Because I bumped the lr for text encoder to 0.9 and after training it didn't fry the model. Plus the generated image looked the same with that from another run, which the lr for text encoder is set to 5e-5. So the text encoder just didn't train.

I think the issue may come from the parameter "Stop text encoder training". The script will ONLY run if this parameter is set to "0", meaning it stops training the text encoder at step 0. 

If I set it to something like 100, the console will throw an error saying "error: unrecognized arguments: --stop_text_encoder_training=100". The script will only run when "Stop text encoder training" is at "0".

Also, I've noticed in one of the earlier patch notes that you can "--train_text_encoder --learning_rate 1e-6: train U-Net and two Text Encoders with the same learning rate (same as the previous version)"

So how do I set "--train_text_encoder" in the GUI? Or do I need to edit the config?

I'm really new to the Kohya script so help is very appreciated :)

**Comment by giteeeeee** [comment]
I'm training with the latest v22.5.0 release, using the gui

**Comment by coreynguyen** [comment]
wow indeed to enable training of the text encoders you need to add '--train_text_encoder' in the 'Additional Parameters' field under the Advanced tab. Thanks for pointing this out, this whole time I thought I was training those and wasn't :\

**Comment by giteeeeee** [comment]
> wow indeed to enable training of the text encoders you need to add '--train_text_encoder' in the 'Additional Parameters' field under the Advanced tab. Thanks for pointing this out, this whole time I thought I was training those and wasn't :\

Thank you instead as I didn't know how to parse "train text encoder" into that additional parameter tab D:

**Comment by giteeeeee** [comment]
I started a new issue https://github.com/bmaltais/kohya_ss/issues/1892

hopefully that will clear things up better
